### PR TITLE
Fixing bugs - econews page

### PR DESCRIPTION
**Pull request by KimAT25**
fixing bugs:
- gallery view,
- delete share btn,
- change request #538,
- updated EcoNews Model file according to new back-end model;

### CODE REVIEW

File: src/app/component/eco-news/news-list/news-list-gallery-view/news-list-gallery-view.component.css [hunk 1]
```diff
@@ -56,7 +56,7 @@ div {
   flex-wrap: wrap;
```
Comment (nabermi2020) [inline src/app/component/eco-news/news-list/news-list-gallery-view/news-list-gallery-view.component.css:56]: After opening 'eco-news' dashboard, scroll to the screen bottom and after that scroll back to the screen top
image

Comment (nabermi2020) [reply src/app/component/eco-news/news-list/news-list-gallery-view/news-list-gallery-view.component.css:56]: Please fix it

Comment (KimAT25) [reply src/app/component/eco-news/news-list/news-list-gallery-view/news-list-gallery-view.component.css:56]: Fixed it. Put in wrong class break avoid

File: src/app/component/eco-news/news-list/news-list-gallery-view/news-list-gallery-view.component.html [hunk 2]
```diff
@@ -17,27 +17,22 @@
   </div>
   <div class="list-text">
     <p>
-      {{ ecoNewsModel.text | slice: 0:200 }}
+      {{ (ecoNewsModel.text.length > 20)?
```
Comment (nabermi2020) [inline src/app/component/eco-news/news-list/news-list-gallery-view/news-list-gallery-view.component.html:20]: Please, create an appropriate method in the component's controller and call it

**Pull request merged**